src/openagi/planner/task_decomposer.py

```diff
@@ -3,6 +3,8 @@
 import re
 from typing import Dict, List, Optional, Union
 
+from click import prompt
```

Comment:
is the requirements.txt required to be updated? @deepakachu-aiplanet

Reply:
No new libraries used, that import was probably a mistake, it wasn't used anyway so i have removed.

all changes needed for cli tool have been made into .toml, env vars also correctly configured.